src/bento/adminData.js

```diff
@@ -28,7 +28,10 @@ export const nodeName = 'Arm(s)'; // Node Name configured in the Data Access Req
 
 export const nodeField = 'numberOfArms'; // Node field
 
+export const commentField = 'Required'; // Comment Field configured as "Required" for Access Pop-up
```

Comment:
According to the user story commentField default is optional.
I would suggest to define a separate variable to control reject btn.
Here, the commentField setting here impacts both approve and reject btn, which may cause some issue in the future. It is good to have control on individual .